attr/_make.py

```diff
@@ -81,6 +82,12 @@ def attr(default=NOTHING, validator=None,
 
     :param init: Include this attribute in the generated ``__init__`` method.
     :type init: bool
+
+    :param callable convert: :func:`callable` that is called by
```

Comment:
aw I forgot about this syntax! could you convert the remaining arguments to it too for consistency please?